query/schema/Orders.js

```diff
@@ -0,0 +1,31 @@
+cube(`Orders`, {
+  sql: `
+  select 1 as id, 100 as amount, 'new' status
+  UNION ALL
+  select 2 as id, 200 as amount, 'new' status
+  UNION ALL
+  select 3 as id, 300 as amount, 'processed' status
```

Comment:
please remove this file, our system should be generic enough to create new schemas on the fly. Move this to tests directory and write tests.